ios/ixoWallet.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
+	<true/>
+</dict>
+</plist>
```

Comment:
Again, don't really care but just curious. Any ideas about what this file is?

Reply:
After some small research, it is a new config file to track env requirement for Xcode IDE.
![image](https://user-images.githubusercontent.com/11903062/108904183-40f0de80-762f-11eb-8cc4-85f59f740283.png)
https://developer.apple.com/library/archive/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html#//apple_ref/doc/uid/TP40001051-CH1-DontLinkElementID_24

Reply:
wonderful thanks for the info